src/utils/fusionAuth.ts

```diff
@@ -0,0 +1,150 @@
+import fs from 'fs-extra';
```

Comment:
I think the types of this file are off. Most function have a return type of `Promise<response>` while probably they all return something different.